setuptools_odoo/make_default_setup.py

```diff
@@ -120,6 +121,62 @@ def make_default_setup_addons_dir(addons_dir, force,
                                  odoo_version_override)
 
 
+def clean_setup_addons_dir(addons_dir):
+    paths_to_remove = []
+
+    addons_setup_dir = os.path.join(addons_dir, 'setup')
+    for addon_name in os.listdir(addons_setup_dir):
+        addon_setup_dir = os.path.join(addons_setup_dir, addon_name)
+        is_directory = os.path.isdir(addon_setup_dir)
+        skip_entry = not is_directory or addon_name == '_metapackage'
```

Comment:
is_setup_dir =  
  exists(addon_setup_dir, 'setup.py') and
  (islink(addon_setup_dir/odoo_addons) or islink(addon_setup_dir/odoo/addons))

We never know what additional directories people may want to add.

Reply:
Do you mean 
```python
islink(addon_setup_dir/odoo_addons/<ADDON_NAME>
or islink(addon_setup_dir/odoo/addons/<ADDON_NAME>))
``` ?

Reply:
ah, yes